pandas/tests/indexes/test_interval.py

```diff
@@ -327,171 +335,298 @@ def test_get_item(self):
                                              closed='right')
         tm.assert_index_equal(result, expected)
 
-    def test_get_loc_value(self):
```

Comment:
to the extent you are adding *new* tests that are xfailing, but let's these these where they are for now.

actually let's put all of your new tests in a new file, its going to be much more obvious. This way all existing behavior is preserved (as we have not changed anything).

You can certainly mark existing tests (maybe with a comment), that this is going to change.

Reply:
That was the way I had it before, but @jorisvandenbossche (I believe it was you?) asked that I remove the current tests instead. I'm happy to go back to what I had before, but please settle this amongst yourselves so we don't revert a third time.

Reply:
it was not me who asked, but that doesn't matter anymore. I think we agree now to go this route. I don't think you should 'go back to what you had before', I think you should be able to just revert that single commit where you removed them to get them back.